zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -1397,7 +1397,7 @@ private void checkpointNote(NotebookSocket conn, Notebook notebook,
     String commitMessage = (String) fromMessage.get("commitMessage");
     AuthenticationInfo subject = new AuthenticationInfo(fromMessage.principal);
     Revision revision = notebook.checkpointNote(noteId, commitMessage, subject);
-    if (revision != null) {
+    if (!Revision.EMPTY.equals(revision)) {
```

Comment:
@anthonycorbacho yeah right, addressed in 13178aee18c0a4aed1dda0c72750e08c9ed9f53c